CONTRIBUTING.md

```diff
@@ -2,7 +2,9 @@
 
 We welcome contributions to the project in any form, including bug reports, bug fixes, new features, improved documentation, or improved test coverage.
 
-Developer-specific documentation is available at https://roualdes.github.io/bridgestan/latest/internals.html
+**Developer-specific documentation is available at [as part of our online documentation](https://roualdes.github.io/bridgestan/latest/internals.html)**. Information on building BridgeStan, running tests, and developing with the project is available there.
+
+This document houses additional information about licensing and procedures for contributing to the project.
```

Comment:
houses -> contains OR supplies OR provides, because there's no housing, so it's distracting.